dev/io.openliberty.microprofile.health.file.healthcheck.internal_fat/fat/src/io/openliberty/microprofile/health/file/healthcheck/fat/LongCheckIntervalHealthCheckTest.java

```diff
@@ -0,0 +1,140 @@
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.health.file.healthcheck.fat;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.time.Duration;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.MicroProfileActions;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.microprofile.health.file.healthcheck.fat.utils.Constants;
+import io.openliberty.microprofile.health.file.healthcheck.fat.utils.HealthFileUtils;
+
+@RunWith(FATRunner.class)
+public class LongCheckIntervalHealthCheckTest extends BaseHealthFilesTest {
+
+    private static final String FAIL_START_APP = "FailStartApp";
+    private static final String FAIL_START_APP_WAR = FAIL_START_APP + ".war";
+
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(
+            FeatureReplacementAction.ALL_SERVERS,
+            MicroProfileActions.MP61,
+            MicroProfileActions.MP70_EE10,
+            MicroProfileActions.MP70_EE11,
+            io.openliberty.microprofile.health.internal_fat.shared.HealthActions.MP14_MPHEALTH40,
+            io.openliberty.microprofile.health.internal_fat.shared.HealthActions.MP41_MPHEALTH40);
+
+    @Server(SERVER_LONG_CHECK_INTERVAL)
+    public static LibertyServer serverLongCheck;
+
+    private static final String[] IGNORED_FAILURES = { "CWMMH0052W", "CWMMH0054W", "CWMMH0053W", "CWMMH0050E" };
+
+    @Before
+    public void before() throws Exception {
+        if (serverLongCheck != null) {
+            serverLongCheck.removeAllInstalledAppsForValidation();
+            serverLongCheck.deleteAllDropinApplications();
+            if (serverLongCheck.isStarted()) {
+                serverLongCheck.stopServer(IGNORED_FAILURES);
+            }
+        }
+    }
+
+    @After
+    public void after() throws Exception {
+        if (serverLongCheck != null && serverLongCheck.isStarted()) {
+            serverLongCheck.stopServer(IGNORED_FAILURES);
+        }
+    }
+
+    /** sample lastModified twice until it stops moving (short cap) */
```

Comment:
We're waiting in milliseconds, why get nano seconds when we get get system time in millis.
This method at best just checks if the file doesn't update within 500 ms. And assumes it may keep updating constantly for 6 seconds? (why 6?).